src/components/UserProfile/UserProfileEdit/UserProfileEdit.jsx

```diff
@@ -0,0 +1,641 @@
+import React, { Component } from 'react'
+import Loading from '../../common/Loading'
+import {
+	Row,
+	Label,
+	Input,
+	CardTitle,
+	Col,
+	Container,
+	Button,
+	Badge,
+	Collapse,
+} from 'reactstrap'
+
+import Image from 'react-bootstrap/Image'
+import { orange, warningRed } from '../../../constants/colors'
+import BlueSquare from '../BlueSquares'
+import Modal from '../UserProfileModal'
+import UserLinks from '../UserLinks'
+import ToggleSwitch from './ToggleSwitch'
+
+import styleProfile from '../UserProfile.css'
+import styleEdit from './UserProfileEdit.css'
+
+
+class EditProfile extends Component {
```

Comment:
updateUserProfile is not updating the job title when it is changed.

Reply:
Fixed with running build on the backend